package-testing/pre/000_install_package.rb

```diff
@@ -1,18 +1,42 @@
+require 'open-uri'
+require 'json'
+
 test_name 'Install pdk package on workstation host' do
   workstation = find_at_most_one('workstation')
 
   step 'Install pdk package' do
     if ENV['LOCAL_PKG']
       pkg = File.basename(ENV['LOCAL_PKG'])
       scp_to(workstation, ENV['LOCAL_PKG'], pkg)
-    end
+    else
+      metadata_url = "http://#{ENV['BUILD_SERVER']}/pdk/#{ENV['SHA']}/artifacts/#{ENV['SHA']}.build_metadata.json"
+
+      begin
+        build_info = JSON.parse(open(metadata_url).read)
+      rescue StandardError => e
+        abort("Could not get build metadata from build server. Tried to request '#{metadata_url}' and got: #{e.message}")
+      end
 
-    if workstation['platform'] =~ %r{windows}
-      pkg ||= "http://#{ENV['BUILD_SERVER']}/pdk/#{ENV['SHA']}/repos/windows/pdk-x64.msi"
+      full_version = build_info['version']
```

Comment:
It might be most helpful to use SUITE_VERSION if it's already set; and fall back to build_metadata.json if it's not set.

I'm unsure if I prefer the code being here or in the Rakefile. It might be better in the Rakefile in order to ensure we have all the essential data before executing beaker and allocating VMs